src/organization-domains/interfaces/create-organization-domain-options.interface.ts

```diff
@@ -0,0 +1,9 @@
+export interface CreateOrganizationDomainOptions {
+  domain: string;
+  organizationId: string;
+}
+
+export interface SerializedCreateOrganizationDomainOptions {
+  domain: string;
+  organization_id: string;
```

Comment:
I see that here we're passing `organization_id` snake case and `organizationId` camel case on `CreateOrganizationDomainOptions`

I presume the Node SDK follows a camel case pattern?